vulnerablecode/settings.py

```diff
@@ -150,11 +150,23 @@
 
 LOGIN_REDIRECT_URL = "/"
 LOGOUT_REDIRECT_URL = "/"
-THROTTLING_RATE = env.str("THROTTLING_RATE", default="1000/day")
+TEST_PACKAGE_THROTTLING_RATE = None
+TEST_BULK_SEARCH_PACKAGE_THROTTLING_RATE = None
+TEST_ALL_VULNERABLE_PACKAGE_THROTTLING_RATE = None
+TEST_VULNERABILITIES_THROTTLING_RATE = None
+TEST_CPES_THROTTLING_RATE = None
+TEST_BULK_SEARCH_CPES_THROTTLING_RATE = None
+TEST_ALIASES_THROTTLING_RATE = None
```

Comment:
This setup is too complex. My suggestion:

```
REST_FRAMEWORK_DEFAULT_THROTTLE_RATES = {
    "vulnerable_packages": "1/hour",
    "bulk_search_packages": "5/hour",
    "packages": "10/minute",
    "vulnerabilities": "10/minute",
    "aliases": "5/minute",
    "cpes": "5/minute",
    "bulk_search_cpes": "5/hour",
}

if IS_TESTS:
    REST_FRAMEWORK_DEFAULT_THROTTLE_RATES = {
        "vulnerable_packages": "1/day",
        "bulk_search_packages": "6/day",
        "packages": "10/day",
        "vulnerabilities": "8/day",
        "aliases": "2/day",
        "cpes": "4/day",
        "bulk_search_cpes": "5/day",
    }

REST_FRAMEWORK = {
  ...,
  "DEFAULT_THROTTLE_RATES": REST_FRAMEWORK_DEFAULT_THROTTLE_RATES,
  ...,
}

```

Reply:
Done!